packages/app/screens/user/Settings.tsx

```diff
@@ -122,15 +122,15 @@ export default function Settings() {
                   <RLabel>Weather: </RLabel>
                   <FormSelect
                     options={weatherOptions}
-                    name="preferredWeather"
+                    name="value.preferredWeather"
```

Comment:
Are we nesting object or some kind of deep linking of form field names to values? If not I believe the first should be okay.

Reply:
you are correct.. we aren't nesting it so i reverted it also